src/views/update-password.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+  <ion-page>
+    <ion-content :fullscreen="true">
+      <div class="flex">
+        <!-- TODO Handle enter key and prevent form submission -->
+        <v-form
+          :validation-schema="validationSchema"
+          :initialValues="formData"
+          class="update-password-container"
+          v-slot="{ validate, errors, resetForm, values }"
+        >
+          <img src="../assets/images/hc.png" />
```

Comment:
Use Image component instead.

Reply:
Image component isn't working here, with this image